doc/args.rst

```diff
@@ -1,13 +1,12 @@
-Command line
-------------
+Parametrizing execution
```

Comment:
"Parametrizing execution" is really weird title

Reply:
Well this is an article about execution parameters, and nearby articles are 'installation' and 'configuring'. Do you have better ideas?

Reply:
"Command line" is perfectly fine.